_includes/share.html

```diff
@@ -1,7 +1,13 @@
-<section class="share">
-  <div class="share-title">Share this article</div>
-  <p>Did this article help you? Share it to inform others.</p>
-  <a class="btn btn-sm twitter" href="https://twitter.com/intent/tweet?text={{ page.title }}&url=https://runnable.com/docs{{ page.url }}" target="_blank">Share to Twitter</a>
-  <a class="btn btn-sm facebook" href="https://www.facebook.com/sharer/sharer.php?u=https://runnable.com/docs{{ page.url }}" target="_blank">Share to Facebook</a>
-  <a class="btn btn-sm linkedin" href="https://www.linkedin.com/shareArticle?mini=true&url=https://runnable.com/docs{{ page.url }}" target="_blank">Share to Linkedin</a>
+<section class="share align-center">
+  <div class="grid-block shrink align-center justify-center">
+    <a class="share-icons" href="https://twitter.com/intent/tweet?text={{ page.title }}&url=https://runnable.com/docs{{ page.url }}" target="_blank">
+      <img class="img" height="32" width="32" src="{{ site.baseurl }}/images/icons/icons-twitter.svg"></img>
+    </a>
+    <a class="share-icons" href="https://www.facebook.com/sharer/sharer.php?u=https://runnable.com/docs{{ page.url }}" target="_blank">
+      <img class="img" height="32" width="32" src="{{ site.baseurl }}/images/icons/icons-facebook.svg"></img>
+    </a>
+    <a class="share-icons" href="https://www.linkedin.com/shareArticle?mini=true&url=https://runnable.com/docs{{ page.url }}" target="_blank">
+      <img class="img" height="48" width="48" src="{{ site.baseurl }}/images/icons/icons-linkedin.svg"></img>
```

Comment:
I think the linked in logo is actually slightly different from what we have now. https://brand.linkedin.com/visual-identity/logo